en/04_Changelogs/5.0.0.md

```diff
@@ -203,6 +204,47 @@ their aliases (applied by setting a string key for the array) were being ignored
 - PHPUnit 5.7 is no longer supported. It is recommended that you ensure your tests are running with PHPUnit 9 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/phpunit9/).
 - `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The optional module `silverstripe/html5` is no longer required for rendering HTML 5 content.
 
+### URLs generated by Silverstripe CMS don't include a trailing slash by default {#trailing-slash}
+
+The URLs generated by Silverstripe CMS have previously been inconsistent as to whether a trailing slash is included or not. This lead to the creation of [at least one module](https://github.com/axllent/silverstripe-trailing-slash) which normalises the trailing slash at the end of URLs.
+
+By default, all URLs created by Silverstripe will now _not_ end with a slash. You can configure this to instead explicitly add a trailing slash to all URLs with the following YML configuration:
+
+```yml
+SilverStripe\Control\Controller:
+    add_trailing_slash: true
+```
+
+[warning]
+Because this can be controlled with configuration, it is best practice to avoid explicitly expecting a trailing slash to either be present or be omitted.
+
+In PHP you can use methods like [Controller::join_links()](api:SilverStripe\Control\Controller::join_links()) or [Controller::normaliseTrailingSlash()](api:SilverStripe\Control\Controller::normaliseTrailingSlash()). In JavaScript in your CMS customisations, we recommend using using the `joinUrlPaths()` utility function from [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config). In your templates, you should use appropriate methods from your controller or model such as [SiteTree::Link()](api:SilverStripe\CMS\Model\SiteTree::Link()) which uses `Controller::join_links()` under the hood.
+[/warning]
+
+The [CanonicalURLMiddleware](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) will also, by default, redirect traffic to include or omit the trailing slash according to the above configuration. By default, this means that traffic directed to `/about-us/` will be redirected to `/about-us`. You can configure this with the following YML configuration:
+
+```yml
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Control\Middleware\CanonicalURLMiddleware:
+    properties:
+      forceTrailingSlash: false
```

Comment:
I've recommended we rename the property because it's very confusing - https://github.com/silverstripe/silverstripe-framework/pull/10538/files#r1080596063

Reply:
done